config.yml

```diff
@@ -83,6 +83,7 @@ development:
   azure:
     account: jungle
   app:
+    awsInstancePubkey: 'fake-pubkey'
```

Comment:
The field is required, but if you're not running the provisioner, it's not actually used.  So this gets the web server to run without having to create a key.